api/v3/Relationship.php

```diff
@@ -146,3 +146,17 @@ function _civicrm_api3_handle_relationship_type(&$params) {
     }
   }
 }
+
```

Comment:
This fix looks good. Can you add a comment block for this FN?

FYI there's a cool new feature in the api explorer - check the Code Docs tab and it displays the comment blocks from all api functions in a readable format.